test/unit/lib/mapLink.js

```diff
@@ -85,6 +85,57 @@ describe('mapLink', () => {
       expect(results[0].mapUrl).to.be.equal(expectedMapLink);
     });
 
+    it('should leave start address empty for place searches', () => {
```

Comment:
Just slightly unclear as to the difference between this unit test and the one which has also been added below below without digging into the code.

Reply:
Thanks, the other one should have said postcode, so should be clearer now.